src/amulware.Graphics/Core/Color.cs

```diff
@@ -363,42 +363,42 @@ public Color(Color color, byte newAlpha)
         /// <returns>The constructed colour.</returns>
         public static Color FromHSVA(float h, float s, float v, byte a = 255)
```

Comment:
Should we spell out the parameter names here?

Reply:
Done. Then again, not sure how far we want to go. Should we spell out `r, g, b, a` variables as well?

Reply:
In public interfaces perhaps? Then again, rgb is very obvious to me. everything else potentially less so.